src/App.tsx

```diff
@@ -1,10 +1,42 @@
-import React from 'react';
+import React, { FC } from 'react';
+
+import { connect } from 'react-redux';
+
+import UserList from './components/UserList/UserList';
 import './App.css';
+import { loadUsers } from './store/store';
+
+const App: FC<any> = ({ prepearedTodoList, loadingCondition, loadUsers }) => {
+  return (
+    <>
+      {
+        prepearedTodoList.length === 0
+          ? (
+            <button
+              type="button"
+              onClick={loadUsers}
+              disabled={loadingCondition}
+              className="load-btn"
+            >
+                    Load
+            </button>
+          )
+          : <UserList prepearedTodoList={prepearedTodoList} />
+      }
+      {loadingCondition && (
+        <p className="loading-text">Loading ...</p>
+      )}
+    </>
+  );
+};
+
+const mapStateToProps = (state: InitialStateInterface) => ({
+  loadingCondition: state.loadingCondition,
```

Comment:
уже наверное тридцатый раз корректирую название этой переменной
```suggestion
  isLoading: state.isLoading,
```